### PR TITLE
[stable/drone] add configurable tolerations for server

### LINKED ISSUE #14581: [stable/drone] add configurable tolerations for server
**Is your feature request related to a problem? Please describe.**
Support configurable tolerations

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by omegazeng**
#### What this PR does / why we need it:
[stable/drone] add support server.tolerations
update values.yaml
update readme.md
#### Which issue this PR fixes
  - fixes #14581

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/drone/values.yaml [hunk 1]
```diff
@@ -248,6 +253,11 @@ agent:
   ##
   nodeSelector: {}
 
+  ## Node taints to tolerate
```
Comment (zakkg3) [inline stable/drone/values.yaml:256]: Why not add tolerations in templates/deployment-agent.yaml if you include this in the agent values?

**Review by zakkg3 — CHANGES_REQUESTED** [review]

File: stable/drone/values.yaml (hunk 1 above)
Comment (omegazeng) [reply stable/drone/values.yaml:256]: https://github.com/helm/charts/blob/master/stable/drone/templates/deployment-agent.yaml#L37
already added, but the values.yaml is not updated.

Comment (zakkg3) [reply stable/drone/values.yaml:256]: fair enough :)

**Pull request merged**